Makefile

```diff
@@ -18,7 +18,7 @@ default:
 venv: $(VENV_ACTIVATE)
 
 $(VENV_ACTIVATE): requirements*.txt setup.py
-	test -f $@ || virtualenv --python=python2.7 $(VENV_DIR)
```

Comment:
This change was so I could run the tests on my Mac. The tests wouldn't run properly under 2.7 and python3.6 is no longer installable via homebrew.